documentation/examples/21-miscellaneous/02-immutable-data/ImmutableTodo.svelte

```diff
@@ -15,7 +15,8 @@
 
 <!-- the text will flash red whenever
 		the `todo` object changes -->
-<div bind:this={div} on:click>
+<!-- svelte-ignore a11y-click-events-have-key-events -->
+<div bind:this={div} on:click role="button" tabindex='0'>
```

Comment:
Instead of ignoring this warning, we should add a button instead that will complete the todo and attach the click handler to that